src/lib/views/account-settings/LeaveRally.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
+  /* This Source Code Form is subject to the terms of the Mozilla Public
+   * License, v. 2.0. If a copy of the MPL was not distributed with this
+   * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+  import Button from "../../../lib/components/Button.svelte";
+  import "../../../lib/components/auth-cards/Auth.css";
+</script>
+
+<div class="leave-rally-wrapper">
+  <h5 class="setting-title">Before You Go…</h5>
+  <div
+    class="split-content-modal"
+    style="margin-bottom: 24px; box-sizing: content-box;"
+  >
+    <div style="width: 368px;">
+      <p class="lead" style="padding-top: 20px;">
```

Comment:
Can we get ride of the line styles?

Reply:
Yup, forgot to clean it up when I set this up. Some of the code from before I came has inline styles that I have to move over so this will remind me to do that.